Geometry/HGCalCommonData/src/HGCalGeomParameters.cc

```diff
@@ -92,7 +92,7 @@ void HGCalGeomParameters::loadGeometryHexagon(const DDFilteredView& _fv,
       double yy = HGCalParameters::k_ScaleFromDDD * fv.translation().Y();
       if (std::abs(yy) < tolerance)
         yy = 0;
-      const CLHEP::Hep3Vector h3v(xx, yy, fv.translation().Z());
+      const CLHEP::Hep3Vector h3v(xx, yy, zp);
```

Comment:
CMSSW is moving away from CLHEP. It would be good to replace `CLHEP::Hep3Vector` with the ROOT version, probably from here:
```
#include "DataFormats/Math/interface/Vector3D.h"
```

Reply:
Plan to revise Geometry/CaloGeometry and then I shall change geometries of Hcal and HGCal. I have that in mind. But I cannot do it now.